### PR TITLE
[TA-4816] Display single row of events. fit as many as possible.

### PR DESCRIPTION
**Pull request by ryanbooker**

### CODE REVIEW

File: src/components/events/Event/index.js [hunk 1]
```diff
@@ -11,15 +11,16 @@ module.exports = React.createClass({
...
     backgroundImageUrl: React.PropTypes.string.isRequired,
     supporterCount: React.PropTypes.number.isRequired,
+    width: React.PropTypes.string.isRequired,
     i18n: React.PropTypes.object,
   },
 
   getDefaultProps: function() {
     return {
       defaultI18n: {
+        backgroundColor: '#525252',
```
Comment (benoneal) [inline src/components/events/Event/index.js]: This doesn't belong in the internationalisation. Just leave it as a separate prop.

File: src/components/events/Event/index.js [hunk 2]
```diff
@@ -64,37 +65,52 @@ module.exports = React.createClass({
...
           <ul className="Event__date">
             <li>{ date.getDate() }</li>
-            <li>{ this.t('months')[date.getMonth()] }</li>
+            <li>{ t('months')[date.getMonth()] }</li>
             <li>{ date.getFullYear() }</li>
           </ul>
-          <p className="Event__name">{ this.props.name }</p>
-          <p className="Event__supporter-count">{ (this.props.supporterCount || 0) + ' ' + this.t('supportersLabel') }</p>
-          <a href={ this.props.getStartedUrl } className="Event__join-event">{ this.t('joinLabel') }</a>
+          <p className="Event__name">{ props.name }</p>
```
Comment (benoneal) [inline src/components/events/Event/index.js]: This should link to the campaign's microsite.

File: src/components/events/UpcomingEvents/index.js [hunk 3]
```diff
@@ -27,23 +30,41 @@ module.exports = React.createClass({
...
 
+  componentDidMount: function() {
+    addEventListener(window, 'resize', this.resize);
+  },
+
+  componentWillUnmount: function() {
+    removeEventListener(window, 'resize', this.resize);
+  },
+
+  resize: function() {
```
Comment (benoneal) [inline src/components/events/UpcomingEvents/index.js]: Debounce this method: resize: 
```_.debounce(function() {}, 100, { trailing: true }),```

File: src/components/events/UpcomingEvents/style.scss [hunk 4]
```diff
@@ -1 +1,6 @@
-.UpcomingEvents { line-height: 0; }
+.UpcomingEvents {
+  line-height: 0;
+  -moz-box-shadow: -1px 1px 3px 0px rgba(0, 0, 0, 0.2);
+  -webkit-box-shadow: -1px 1px 3px 0px rgba(0, 0, 0, 0.2);
```
Comment (benoneal) [inline src/components/events/UpcomingEvents/style.scss]: These vendor prefixes aren't needed.

File: src/components/events/Event/index.js (hunk 1 above)
Comment (ryanbooker) [inline src/components/events/Event/index.js]: Whoops put it in the wrong block.

File: src/components/events/Event/index.js (hunk 2 above)
Comment (ryanbooker) [inline src/components/events/Event/index.js]: Is there already a function that returns this?

Comment (benoneal) [inline src/components/events/Event/index.js]: Just make it an anchor tag with a href.

Comment (ryanbooker) [inline src/components/events/Event/index.js]: I mean to work out what the link is. It's not in the data you get back, and as someone who hasn't been here for year I have no idea what it is off the top of my head. :)

**Pull request merged**